### ISSUE TITLE
user portal

### ISSUE THREAD

**Issue by eatyourpeas** [rcpch/national-paediatric-diabetes-audit#10]
Allied to #7 
Need a user portal similar to E12
Clinician leads for each PDU would have powers to create users in their PDU only. RCPCH audit leads have national oversight.
There needs to be an audit trail.
It is likely we can reuse a lot of E12 learning / code

**Comment by eatyourpeas** [comment]
Largely implemented with #17

**Comment by mbarton** [comment]
> Largely implemented with https://github.com/rcpch/national-paediatric-diabetes-audit/pull/17

@eatyourpeas have we got issues to track what's not been implemented?

**Comment by eatyourpeas** [comment]
sorry yes - partly this issue overlaps with:
- organisations api implementation - each user is affiliated with an organisation so this relationship will need including in the User model. I have commented it out at the moment
- permissions and groups - these have now been implemented but I don't fully Holly's document so I have guessed a little what I think she wants here. For example she requests that certain users have data 'access' but I am not sure if that means view or view and update. It is likely we will need to add and change some of the permissions I have created
- email sign/up/account creation pathway: I am working on this now in a separate branch
- user log
- security decorators/mixins: Django has these that protect routes against access by those who are not logged in. We need to create some new ones though to scope access to group, so that those associated with an organisation, can only see others in the same trust, and those with RCPCH status can see nationally.
